crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -219,6 +219,10 @@ def families
         })
       end
     end
+
+    respond_to do |format|
+      format.html { render "families" }
```

Comment:
Why? You can skip the render. Conventional you want to write just ```format.html``` and nothing more in that case.